src/components/Cart.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+
+export class Cart extends Component {
+    render() {
+        return (
+            <div>
+                <h1>Cart:</h1>
+                <p>Item added, $ x.xx</p>
+                <p>Item added, $ x.xx</p>
+                <p>Item added, $ x.xx</p>
+                <p>Item added, $ x.xx</p>
+
+                <h4>Subtotal:, $ x.xx</h4>
+                <h4>Tax: $ x.xx</h4>
+                <h4>Total: $ x.xx</h4>
```

Comment:
I like that you used placeholder values here. If you were to replace the x.xx values with real variables, where would they come from? Do you already have access to that data?